cms/articles/tests/test_models.py

```diff
@@ -233,26 +234,203 @@ def test_cite_this_page_is_not_shown_when_unticked(self):
+                release_count += 1
+        self.assertEqual(release_count, self.total_batch)
+
+    def test_breadcrumb_is_shown(self):
+        # ToDo
+        # this should include breadcrumb content comparison with the url but there is a bug
+        # mock_url = response.request['PATH_INFO']
+        # mock_url = mock_url.split("/")
+        # mock_url = list(filter(('').__ne__, mock_url))
+        # mock_breadcrumb = self.get_breadcrumbs(response)
+        # self.assertContains(response, expected)
+        # self.assertEqual(len(mock_url) - len(mock_breadcrumb),1)
+        # self.assertEqual(mock_url[0] , mock_breadcrumb[0])
+        # self.assertEqual(mock_url[1] , mock_breadcrumb[1])
+        response = self.client.get(self.previous_releases_url)
+        expected = 'class="ons-breadcrumbs__link"'
+        self.assertContains(response, expected)
+
+    def test_pagination_is_not_shown(self):
+        response = self.client.get(self.previous_releases_url)
+        expected = 'class="ons-pagination__link"'
+        self.assertNotContains(response, expected)
+
+
+@override_settings(PREVIOUS_RELEASES_PER_PAGE=3)
+class PreviousReleasesWithPaginationPage3TestCase(TestCase):
+    total_batch = 13
+    current_page_number = 3
+    PREVIOUS_RELEASES_PER_PAGE = 3
+    total_no_of_pages = math.ceil(total_batch / PREVIOUS_RELEASES_PER_PAGE)
+
+    @classmethod
+    def setUpTestData(cls, cls_total_batch: int = total_batch, cls_current_page_number: int = current_page_number):
+        cls.article_series = ArticleSeriesPageFactory(title="Article Series")
+        cls.articles = StatisticalArticlePageFactory.create_batch(cls_total_batch, parent=cls.article_series)
+        cls.previous_releases_url = (
+            cls.article_series.url
+            + cls.article_series.reverse_subpage("previous_releases")
+            + f"?page={cls_current_page_number}"
+        )
+
+    def test_pagination_is_shown(self):
+        response = self.client.get(self.previous_releases_url)
+        expected = [
+            f'class="ons-pagination__position">Page {self.current_page_number} of {
+                math.ceil(self.total_batch / settings.PREVIOUS_RELEASES_PER_PAGE)
+            }',
+            'aria-label="Go to the first page (Page 1)"',
+            'class="ons-pagination__item ons-pagination__item--previous"',
+            'class="ons-pagination__item ons-pagination__item--current"',
+            f'aria-label="Go to the last page (Page {self.total_no_of_pages})"',
+            'class="ons-pagination__item ons-pagination__item--next"',
+        ]
+        not_expected = []
+        for expect in expected:
+            self.assertContains(response, expect)
+        for not_expect in not_expected:
+            self.assertNotContains(response, not_expect)
+
+
+@override_settings(PREVIOUS_RELEASES_PER_PAGE=3)
+class PreviousReleasesWithPaginationPage1TestCase(TestCase):
+    total_batch = 13
+    current_page_number = 1
+    PREVIOUS_RELEASES_PER_PAGE = 3
```

Comment:
```suggestion
```

unused

Reply:
No longer relevant? @zerolab Can we resolve this?

Reply:
Resolving on behalf!